pages/component/edit.tsx

```diff
@@ -40,6 +43,10 @@ export default function Component() {
           setConfig(message.data ?? {});
           return;
         }
+        case "mode": {
+          console.log("The current mode is", message.data);
```

Comment:
Since we're using a `console.log` anyways, could we also include the current component the log is coming from (i.e. "view" or "edit")?